README.md

```diff
@@ -52,21 +52,31 @@ $logProcessor->setLogInfo(new LogInfo(...));
 ```
 
 ## Development
-### AWS resources
-Use the provided `test-cf-stack.json` to create a CloudFormation stack. Use the outputs to set environment variables
-`AWS_DEFAULT_REGION`, `S3_LOGS_BUCKET`. Create an access key for the generated user. Set it to the environment 
-variables `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY`.
 
-### Azure resources
-Use the provided `test-arm-template.json` to create ARM stack:
+Prerequisites:
+* locally installed `terraform` and `jq`
+    * https://www.terraform.io
+    * https://stedolan.github.io/jq
+* configured `az` and `aws` CLI tools (run `az login` and `aws configure --profile YOUR_AWS_PROFILE_NAME`)
 
-    az group create --name testing-api-error-control --location "East US"
+### AWS and Azure resources
```

Comment:
Nitpick, ale v podstate to nejsou "AWS and Azure resources", ale obecne vsechny provisionovany resources. Pak to nekde C&P do projektu, kde bude i K8S a nebude to pravda :slightly_smiling_face: